### PR TITLE
feat(react): added disabled prop to file uploader

### PR DESCRIPTION
**Pull request by TannerS**
Closes #

Not an issue but mentioned here https://ibm-cloudplatform.slack.com/archives/C0M053VPT/p1643384630118300

#### Changelog

**New**

Added disable prop to `FileUploader` and disabled styles to its label description and label text div elements

#### Testing / Reviewing

Verified in storybook, if unit test are needed, any feedback is appreciated

### CODE REVIEW

File: packages/react/src/components/FileUploader/FileUploader-story.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ const props = {
       ),
       buttonLabel: text('The button label (buttonLabel)', 'Add file'),
       buttonKind: buttonKind || 'primary',
+      disabled: boolean('Disabled (disabled)', false),
```
Comment (tay1orjones) [inline packages/react/src/components/FileUploader/FileUploader-story.js:90]: We'll need to mirror this change into the v11 storybook as well through 
 `packages/react/src/components/FileUploader/next/FileUploader.stories.js`. I'll push a quick commit for it

**Review by tay1orjones — APPROVED** [review]

File: packages/react/src/components/FileUploader/FileUploader-story.js (hunk 1 above)
Comment (tay1orjones) [reply packages/react/src/components/FileUploader/FileUploader-story.js:90]: Added!

Comment (TannerS) [reply packages/react/src/components/FileUploader/FileUploader-story.js:90]: thanks! didn't know you guys had a v11 storybook @tay1orjones

**Review by dakahn — CHANGES_REQUESTED** [review]
Im getting a flash of incorrect styling on the dotted box when i mouse out on Firefox latest. Also I'm tapping in a designer here to make sure the styles are above board given we're not working off of a triaged issue ticket.

**Comment by TannerS** [comment]
> Im getting a flash of incorrect styling on the dotted box when i mouse out on Firefox latest. Also I'm tapping in a designer here to make sure the styles are above board given we're not working off of a triaged issue ticket.

Hello, may i know what dotted line you are referring too? and let me know how I can help

**Comment by TannerS** [comment]
@dakahn  ^

**Comment by dakahn** [comment]
@TannerS 
image
the dotted line surrounding the drop zone for files. on mouse out theres a flash of incorrect styling

**Comment by TannerS** [comment]
Hey @dakahn my code should have only touched the file uploader not the drop and drag one unless I made a mistake? I will review if my code somehow affected that however too!

**Comment by dakahn** [comment]
@TannerS nope, you're right. disregard my comment. looks like its a bug in the stable version 🤦🏾

**Review by dakahn — APPROVED** [review]

**Comment by TannerS** [comment]
@dakahn got ya, thanks for that

@tay1orjones I hit re-review by mistake, just a fyi its asking you to re-approval

**Review by laurenmrice — APPROVED** [review]
Looks good, thank you for contributing! ✅

**Pull request merged**